users/README.md

```diff
@@ -0,0 +1,26 @@
+# Users
+
+```
+{
+  'id': string,
+  'username': string,
+  'first_name': string,
+  'last_name': string,
+  'email': string,
+  'phone': number,
+  'yoe': number,
+  'company': string,
+  'designation': string,
+  'img': string,
+  'github_id': string,
+  'linkedin_id': string,
+  'twitter_id': string,
+  'instagram_id': string,
+  'website': string,
+  'github_display_name': string,
+  'isMember': boolean,
+  'userType': string,
+  'tokens': {},
+  'badges': []
```

Comment:
Are you sure we're going to have badges here? In API contract for `badges` I saw that we're having a badges document and maintaining an array of users to which that particular badge has been assigned to.

Reply:
I copied structure from code base. Yes, we have separate badges collection in firestore. Not necessary here. Removing it.